bin/build_image.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# build the portal Docker image and tag accordingly, then push to GCR
```

Comment:
Could you add some code commentary around what calls this `build_image.sh` script?  Searching this file's name as a string returns no other code calling to reference it.

Reply:
I think that the work that will be done as a part of SCP-4494 will clarify this - I haven't completely decided on the exact mechanism.  It will either be a separate Jenkins job (which I've started working on already), or this script will just get added to each existing deployment job.

The clean way is the separate job which then triggers the deployment on a successful build (Jenkins supports this, and could give us completely automated deployments on merges if we wanted).  However, adding a single line to each existing job would be much easier: `bin/build_image.sh && bin/deploy.sh [params]`.